### DISCUSSION TITLE
GCode Preview Not Showing Up in 2.3.57

### DISCUSSION THREAD

**Discussion by djngnear** [supermerill/SuperSlicer#1570] [Q&A]
I did a search here and couldn't find a discussion on this. But I recently updated to SS 2.3.57, and after hitting slice nothing comes up in the gcode preview tab. It says slicing complete in the bottom left corner and the model shows up under the sliced preview tab but nothing under the gcode. Anyone else have this happen to them? SS 2.3.56 doesn't give me this issue so for now that's what I'm gonna use.  

![SS_Capture](https://user-images.githubusercontent.com/87901234/133907278-6c77afe5-7b88-4e7e-88c2-eda1ea0d611e.PNG)

**Comment by supermerill** [comment]
#1532
